arm/avhclient/aws_backend.py

```diff
@@ -820,21 +861,31 @@ def send_remote_command(self, command_list, working_dir, fail_if_unsuccess=True)
         This is a mandatory AVH backend method.
         """
         self._init()
-        logging.info(f"avh: command_list = {command_list}")
+        logging.debug(f"aws:command_list = {command_list}")
         response = self.send_ssm_shell_command(
             command_list=command_list,
             working_dir=working_dir
         )
 
+        log_init = f"{''.join('-' for x in range(80))}"
+        logging.info(log_init) if enable_logging_info else logging.debug(log_init)
```

Comment:
Why do we need different log levels in this case?
```suggestion
        logging.log(logging.INFO if enable_logging_info else logging.DEBUG, '='*80)
```

Reply:
I would like to avoid to print some command details sent to the EC2 (e.g. for preparing image) which does not bring any value to the user and polute the output. Having this flag, enables us to print the details whenever necessary (e.g. output of AVH commands), but we can still see all the output if in DEBUG logging.

Reply:
Year, understood this right after writing the comment.